simpleclient_spring_boot/src/main/java/io/prometheus/client/spring/boot/PrometheusProperties.java

```diff
@@ -0,0 +1,50 @@
+package io.prometheus.client.spring.boot;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * @author pidster
+ */
+@ConfigurationProperties(prefix = "endpoints.prometheus")
+public class PrometheusProperties {
```

Comment:
I'm wondering how  this class is used - I can see it only in `@EnableConfigurationProperties({PrometheusProperties.class})` in `PrometheusAutoConfiguration`, but I don't see class properties used anywhere. For example I'm trying to find how the `path` is used (I assume this is for path in URL), but it is not clear (to me :) how this is applied.